java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory with fast-jar deployment with Sentry extension

**Describe the bug**
Exception while starting a quarkus app in fast-jar deployment mode:
```bash
java -jar ./target/quarkus-app/quarkus-run.jar                                                                                                                                                                              master ●
sept. 22, 2020 10:59:34 AM io.quarkus.application
ERROR: Failed to start application (with profile prod)
java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
        at io.sentry.config.Lookup.<clinit>(Lookup.java:37)
        at io.quarkus.logging.sentry.SentryHandlerValueFactory.create(SentryHandlerValueFactory.java:23)
        at io.quarkus.deployment.steps.SentryProcessor$addSentryLogHandler-1613402751.deploy_0(SentryProcessor$addSentryLogHandler-1613402751.zig:80)
        at io.quarkus.deployment.steps.SentryProcessor$addSentryLogHandler-1613402751.deploy(SentryProcessor$addSentryLogHandler-1613402751.zig:40)
        at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:400)
        at io.quarkus.runtime.Application.start(Application.java:90)
        at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:91)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:61)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:38)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:106)
        at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:29)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:43)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:24)
Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
        ... 17 more

```

Note: No issue without the Sentry extension, I added it because it's part of the stack..

CC @geoand @stuartwdouglas 


**To Reproduce**

Link to a small reproducer:
https://github.com/ia3andy/reproduce-fast-jar

Steps to reproduce the behavior:
1. `mvn clean package -Dquarkus.package.type=fast-jar `
2. `java -jar ./target/quarkus-app/quarkus-run.jar     `



**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: 
```
Darwin MacBook-Pro-de-ia3andy 18.7.0 Darwin Kernel Version 18.7.0: Thu Jun 18 20:50:10 PDT 2020; root:xnu-4903.278.43~1/RELEASE_X86_64 x86_64
```
 - Output of `java -version`: 
```
openjdk version "11.0.1" 2018-10-16
OpenJDK Runtime Environment 18.9 (build 11.0.1+13)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.1+13, mixed mode)
```

 - Quarkus version or git rev: 
`1.8.1.Final`
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
```
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /Users/ia3andy/.m2/wrapper/dists/apache-maven-3.6.3-bin/1iopthnavndlasol9gbrbg6bf2/apache-maven-3.6.3
Java version: 14.0.1, vendor: AdoptOpenJDK, runtime: /Library/Java/JavaVirtualMachines/adoptopenjdk-14.jdk/Contents/Home
Default locale: fr_FR, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
```

--- a/extensions/logging-sentry/runtime/pom.xml
+++ b/extensions/logging-sentry/runtime/pom.xml
@@ -33,6 +33,8 @@
                 <configuration>
                     <parentFirstArtifacts>
                         <parentFirstArtifact>io.sentry:sentry</parentFirstArtifact>
+                        <!-- slf4j is used in static init of io.sentry.config.Lookup so we need it in the same classloader -->
+                        <parentFirstArtifact>org.slf4j:slf4j-api</parentFirstArtifact>
                     </parentFirstArtifacts>
                 </configuration>
             </plugin>

